### DISCUSSION TITLE
已解决/过时但未关闭的 Issues 收集

### DISCUSSION THREAD

**Discussion by Tinhone** [the1812/Bilibili-Evolved#3973] [综合讨论 / General]
虽说是上古遗物，但再次确认还是有必要的

- #92

- #613

- #768

- #890

- #902
└─ Bilibili Evolved V1 已不再受支持 & 无后续反馈 & [v1.12.13](https://github.com/the1812/Bilibili-Evolved/releases/tag/v1.12.13) 已对可能的问题原因 “收起未关注人消息打开时, 自定义顶栏的消息数目仍然包含未关注人” 进行了修复

- #951

- #1052
└─ 在 [#4001](https://github.com/the1812/Bilibili-Evolved/pull/4001) 修复

- #1178
└─ 直播回放已被改为普通视频形式，不再是独立功能

- #1271 
└─ B 站目前已不再提供 720p60 选项

- #1306
└─ 无后续反馈 & B 站已修复

- #1310
└─ 无后续反馈 & B 站已修复

- #1406
└─ Bilibili Evolved V1 已不再受支持 & 无后续反馈 & [v1.12.13](https://github.com/the1812/Bilibili-Evolved/releases/tag/v1.12.13) 已对可能的问题原因 “收起未关注人消息打开时, 自定义顶栏的消息数目仍然包含未关注人” 进行了修复

- #1475

- #1524

- #1531
└─ 问题已不再出现

- #1670

- #1730

- #1745
└─ Bilibili Evolved V1 已不再受支持 & 无后续反馈

- #1837 
└─ 可解决相关问题的功能在 issue 开启前就存在，后续回复中对如何开启功能进行了指引，之后 issue 开启者表示收到

- #1851
└─ 功能已完成

- #1886

- #1970

- #1995

- #2028

- #2034

- #2050

- #2191 
└─ 根据 [反馈1](https://github.com/the1812/Bilibili-Evolved/issues/3990#issuecomment-1427061949) [反馈2](https://github.com/the1812/Bilibili-Evolved/issues/3990#issuecomment-1427785152)，此问题应该已经修复

- #2200

- #2281

- #2342

- #2388 
└─ Bilibili Evolved V1 已不再受支持 & 无后续反馈

- #2418

- #2521

- #2529 
└─ Bilibili Evolved V1 已不再受支持 & 无后续反馈

- #3683

**Comment by Geeyun-JY3** [comment]
主要是破站目前还留着旧版首页，#1052 在旧版首页还是能复现的，也是很奇怪为啥在新版首页和内测首页出来这么久旧版首页还没砍掉。

**Reply by Tinhone** [reply]
对渣机来说新版首页性能实打实的差，砍掉旧版首页那种事不要啊😭

**Comment by the1812** [comment]
![Snipaste_2023-07-24_22-17-23](https://github.com/the1812/Bilibili-Evolved/assets/26504152/5ab0ace8-2ce2-4881-b3bd-e8564b395cf7)
其实一直想整治下的, 但是我太鸽了(
